ui-poc/pages/Friends.py

```
@@ -0,0 +1,350 @@
+import streamlit as st
+import requests
+from datetime import datetime
```

Ruff (0.11.9)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)